bonsai/Cargo.toml

```diff
@@ -9,12 +9,12 @@ homepage = "https://risczero.com/"
 repository = "https://github.com/risc0/risc0/"
 
 [workspace.dependencies]
-bonsai-ethereum-contracts = { path = "ethereum" }
-bonsai-ethereum-relay = { default-features = false, path = "ethereum-relay" }
-bonsai-rest-api-mock = { default-features = false, path = "rest-api-mock" }
-bonsai-sdk = { default-features = false, path = "sdk" }
-risc0-build = { default-features = false, path = "../risc0/build" }
-risc0-zkvm = { default-features = false, path = "../risc0/zkvm" }
+bonsai-ethereum-contracts = { version = "0.1.0", path = "ethereum" }
+bonsai-ethereum-relay = { version = "0.1.0", default-features = false, path = "ethereum-relay" }
+bonsai-rest-api-mock = { version = "0.1.0", default-features = false, path = "rest-api-mock" }
```

Comment:
Now that we plan to publish these, it would be convenient to keep them aligned to whatever `bonsai-sdk` has.